Source/WebCore/css/CSSProperties.json

```diff
@@ -1149,7 +1149,6 @@
                 "upright"
             ],
             "codegen-properties": {
-                "custom": "Value",
                 "high-priority": true,
                 "parser-grammar": "<<values>>"
```

Comment:
I noticed we have instances where we have the values defined in "parser-grammar" like `-webkit-locale` above, and then those where we store them in the values object.

What is the preferred way of doing it now?

Reply:
In general, when the syntax is simple (single keyword), referring to them as `<<values>>` and listing the keywords in `"values"` is preferred.

If the syntax is more complex (2 keywords, keywords or length, etc), we typically don't do so.